src/applications/vaos/new-appointment/components/TypeOfCarePage/index.jsx

```diff
@@ -134,8 +134,7 @@ export class TypeOfCarePage extends React.Component {
             loadingText="Page change in progress"
           />
         </SchemaForm>
-        <TypeOfCareUnavailableModal
-          typeOfCare="Podiatry"
+        <PodiatryAppointmentUnavailableModal
```

Comment:
I generally agree with moving to a Podiatry specific modal, because it's a pattern really only made for the weird state Podiatry is in. But this does mean that we have type of care agnostic Redux state and actions (`showToCUnavailableModal` and `hideTypeOfCareUnavailableModal`) controlling a Podiatry specific modal. Plus the old, now unused `TypeOfCareUnavailableModal` component is still lingering.

I think I would either go ahead and refactor the Redux state/actions to be specific to Podiatry and delete the old component, or make the language update in the ticket in the old `TypeOfCareUnavailableModal` component (it's fine for it to theoretically apply to other components).

Reply:
I had similar ideas, will refactor to Podiatry specific modal and then delete `TypeOfCareUnavailableModal`

Reply:
Should be resolved now